sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -150,6 +298,38 @@ export class Sender {
     sendBatch(messages: SendableMessageInfo[]): Promise<void>;
     }
 
+// @public
+export class ServiceBusAtomManagementClient extends ServiceClient {
+    constructor(connectionString: string, options?: ServiceBusAtomManagementClientOptions);
+    createQueue(queueName: string, queueOptions?: QueueOptions): Promise<CreateQueueResponse>;
+    createRule(topicName: string, subscriptionName: string, ruleName: string, ruleOptions?: RuleOptions): Promise<CreateRuleResponse>;
+    createSubscription(topicName: string, subscriptionName: string, subscriptionOptions?: SubscriptionOptions): Promise<CreateSubscriptionResponse>;
+    createTopic(topicName: string, topicOptions?: TopicOptions): Promise<CreateTopicResponse>;
+    deleteQueue(queueName: string): Promise<DeleteQueueResponse>;
+    deleteRule(topicName: string, subscriptionName: string, ruleName: string): Promise<DeleteRuleResponse>;
+    deleteSubscription(topicName: string, subscriptionName: string): Promise<DeleteSubscriptionResponse>;
+    deleteTopic(topicName: string): Promise<DeleteTopicResponse>;
+    // (undocumented)
+    getDeadLetterPath(queueName: string): string;
```

Comment:
I thought we have this method on queues already. Are the two different?

Reply:
Oh, should be the same - removed it